src/test/java/org/prebid/server/bidder/admixer/AdmixerBidderTest.java

```diff
@@ -303,6 +267,91 @@ public void makeBidsShouldReturnBannerBidIfAudioIsPresentInRequestImp() throws J
                 .containsExactly(BidderBid.of(Bid.builder().impid("123").build(), audio, "USD"));
     }
 
+    @Test
+    public void shouldReturnNullIfBidFloorNullCustomFloorNull() {
```

Comment:
Method you're testing returns `Result<List<HttpRequest<BidRequest>>>`. It doesn't returns null.
Rename to something like `makeHttpRequestsShouldReturnBidFloorAs...` to be in code style with other tests.
PS. Change all other test names below.

Reply:
Maybe name it as `makeHttpRequestsShouldOmitBidFloor...`?